runtime/compiler/optimizer/JProfilingRecompLoopTest.cpp

```diff
@@ -145,14 +155,14 @@ TR_JProfilingRecompLoopTest::perform()
  *                   --------------------------------                                              |
  *                   |treeTops after insertion Point|  <--- remainingCodeBlock                     |
  *                   |from original block           |                                              |
- *                   |...                           |<----------------------------------------------   
+ *                   |...                           |<----------------------------------------------
  *                   --------------------------------
  *    \param comp Current compilation object
  *    \param testLocations RecompilationTestLocation list containing TreeTops after which test are required to be added
- *                         And corresponding to that location, a loop nesting depth 
+ *                         And corresponding to that location, a loop nesting depth
  */
 
-void 
+void
 TR_JProfilingRecompLoopTest::addRecompilationTests(TR::Compilation *comp, RecompilationTestLocationsInfo &testLocations)
```

Comment:
Actually, can I request you to modify this function signature as well? 
We already get Block FrequencyInfo before continuing perform now, so we can pass that. `TR_PersistentProfileInfo` is not used in this function so we do not need it.

Reply:
Addressed in aa47e0f